tools/openshift-ci/case_id_lookup_table.txt

```diff
@@ -0,0 +1,33 @@
+OCP-9567
```

Comment:
we spoke about file name like `cases_ded_4_0.txt`. Something that will show cluster type and version as selection depends on these.